### ISSUE TITLE
Image colorisation

### ISSUE THREAD

**Issue by manojkumar202** [wyhuai/DDNM#7]
Hi 
I'm trying to run this code for colorization. I'm loading the "celeba_hq.ckpt" model on ddnm to convert my grayscale to color. It produces brownish images. How should I train for colorization? I really appreciate any help you can provide.

**Comment by manojkumar202** [comment]
I noticed that you have mentioned to look at the guided diffusion for training. what I'm unable to figure out is how is should I specify the colorization tasks during training(eg: should I change the loss function).

**Comment by wyhuai** [comment]
> Hi I'm trying to run this code for colorization. I'm loading the "celeba_hq.ckpt" model on ddnm to convert my grayscale to color. It produces brownish images. How should I train for colorization? I really appreciate any help you can provide.

Can you show me your input image?

**Comment by manojkumar202** [comment]
Input image
![b_n_w4](https://user-images.githubusercontent.com/88579494/206884043-e10169bd-9bc9-4acc-a414-70c84f7e2728.jpeg)

output image
![2_0](https://user-images.githubusercontent.com/88579494/206884057-048952ff-1a8a-4a46-b604-479a3a79506a.png)

**Comment by manojkumar202** [comment]
Input image 
![b_n_w_3](https://user-images.githubusercontent.com/88579494/206884134-96d36a59-136e-46d1-9e19-542c07c756a8.jpeg)

output image
![4_0](https://user-images.githubusercontent.com/88579494/206884143-5f487fe9-228e-4a03-bce2-bb9ffb5d07ba.png)

**Comment by manojkumar202** [comment]
It would be really great if I can know how to train for colorization specific task

**Comment by wyhuai** [comment]
You should use the imagenet model for these types and try different random seeds for diverse results. e.g., `python main.py --ni --config imagenet_256.yml --path_y imagenet --eta 0.85 --deg "colorization" --sigma_y 0. -i imagenet_colorization`

**Comment by wyhuai** [comment]
> It would be really great if I can know how to train for colorization specific task

We don't need to train for specific tasks... Our colorization results on the front page are generated using the pre-trained ImgaeNet model with classifier guidance in the /hq_demo, but we only provide the SR task yet. Colorization and other tasks will be updated in the hq_demo in the future. For now, you may implement colorization on your own😂.

**Comment by manojkumar202** [comment]
ok. Thank you.
Seems like that is the option left now 😂.
